noetic/distribution.yaml

```
@@ -8494,6 +8494,21 @@ repositories:
       url: https://github.com/septentrio-gnss/septentrio_gnss_driver.git
       version: master
     status: maintained
+  serial:
+    doc:
+      type: git
+      url: https://github.com/wjwwood/serial.git
+      version: master
+    release:
+      tags:
+        release: release/noetic/{package}/{version}
+      url: https://github.com/wjwwood/serial-release.git
+      version: 1.2.1-1
+    source:
+      type: git
+      url: https://github.com/wjwwood/serial.git
+      version: master
+    status: maintained
```

```suggestion
  serial:
    doc:
      type: git
      url: https://github.com/wjwwood/serial.git
      version: main
    release:
      tags:
        release: release/noetic/{package}/{version}
      url: https://github.com/wjwwood/serial-release.git
      version: 1.2.1-1
    source:
      type: git
      url: https://github.com/wjwwood/serial.git
      version: main
    status: maintained
```

It appears that the default branch has been renamed upstream.